apps/server/src/modules/class/repo/classes.repo.spec.ts

```diff
@@ -129,49 +128,50 @@ describe(ClassesRepo.name, () => {
 				};
 			};
 
-			it('should update classes without deleted user', async () => {
-				const { class1, class2, testUser1, testUser2, testUser3 } = await setup();
+			it('should use nativeUpdate', async () => {
+				const { testUser1 } = await setup();
 
-				class1.userIds = [testUser2];
-				class2.userIds = [testUser3];
+				jest.spyOn(em, 'nativeUpdate');
```

Comment:
Should we do a spy? We can validate if it works or not by checking the in memory db data. All other is a internal of the repo method. It should possible to change it, without touching the code.

Reply:
I would like to keep this, as is a way to ensure we use nativeUpdate for performance reason. 
This should prevent re-implement it similar to how it was before.

Reply:
But it result in fragile tests: https://www.youtube.com/watch?v=WUPlTo7nVa4  
It is always a bad idea. But is your decision.